packages/graph/hash_graph/lib/graph/src/store/postgres/context/links.rs

```diff
@@ -103,6 +106,8 @@ pub async fn read_links_by_target(
             FROM links
             JOIN type_ids ON version_id = link_type_version_id
             WHERE target_entity_id = $1
+            -- When reading links by target, the ordering becomes irrelevant, as the target could
+            -- have a variety of link that are meaningless to sort by their order.
```

Comment:
Nit:
```suggestion
            -- have a variety of link that are meaningless to sort by their index.
```

Reply:
Thanks, applied in 2f659d8ca7be3305c953ef55a638a6e1953dfac1